### PR TITLE
Add TPU service and task example

### PR DESCRIPTION
**Pull request by Bihan**

### CODE REVIEW

**Comment by peterschmidt85** [comment]
What about a README file? Do you have an idea of what could be written there?

**Comment by Bihan** [comment]
> What about a README file? Do you have an idea of what could be written there?

@peterschmidt85 Do you mean this [README](https://github.com/dstackai/dstack/blob/master/examples/README.md) file?

**Comment by peterschmidt85** [comment]
@Bihan I mean typically every example we have has a README.md like https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/axolotl/README.md (a detailed one)or https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/qlora/README.md (a short one - didn't have time to write a good one).

**Comment by Bihan** [comment]
> @Bihan I mean typically every example we have has a README.md like https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/axolotl/README.md (a detailed one)or https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/qlora/README.md (a short one - didn't have time to write a good one).

@peterschmidt85 I had added a README.md [here](https://github.com/Bihan/dstack/blob/tpu_examples/examples/deployment/tpu/tgi/README.md) for the TPU/TGI example. I will add more details in this README file.

**Comment by peterschmidt85** [comment]
> > @Bihan I mean typically every example we have has a README.md like https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/axolotl/README.md (a detailed one)or https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/qlora/README.md (a short one - didn't have time to write a good one).
> 
> @peterschmidt85 I had added a README.md [here](https://github.com/Bihan/dstack/blob/tpu_examples/examples/deployment/tpu/tgi/README.md) for the TPU/TGI example. I will add more details in this README file.

Oh, I missed it when reviewed it. If you want to add more details what can we add that could be very useful?

**Comment by Bihan** [comment]
> > > @Bihan I mean typically every example we have has a README.md like https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/axolotl/README.md (a detailed one)or https://github.com/dstackai/dstack/blob/master/examples/fine-tuning/qlora/README.md (a short one - didn't have time to write a good one).
> > 
> > 
> > @peterschmidt85 I had added a README.md [here](https://github.com/Bihan/dstack/blob/tpu_examples/examples/deployment/tpu/tgi/README.md) for the TPU/TGI example. I will add more details in this README file.
> 
> Oh, I missed it when reviewed it. If you want to add more details what can we add that could be very useful?

@peterschmidt85 We can add about the `image huggingface/optimum-tpu:latest`. Also point to  `optimum-tpu` [repo](huggingface/optimum-tpu:latest) for more details. Mention the models (Gemma (2b, 7b), Llama2 (7b) and Llama3 (8b)) supported by it. Finally, we can mention that all TPU Cores are utilized in Inference.

**Comment by peterschmidt85** [comment]
Sounds good it me

Best regards,
Andrey

File: examples/accelerators/tpu/README.md [hunk 1]
```diff
@@ -12,17 +12,53 @@ Below are a few examples on using TPUs for deployment and fine-tuning.
 
 ### Running as a service
 You can use any serving framework, such as vLLM, TGI. Here's an example of a [service](https://dstack.ai/docs/services) that deploys
-Llama 3.1 8B using [vLLM :material-arrow-top-right-thin:{ .external }](https://github.com/huggingface/optimum-tpu){:target="_blank"} or 
-[Optimum TPU :material-arrow-top-right-thin:{ .external }](https://github.com/huggingface/optimum-tpu){:target="_blank"}.
+Llama 3.1 8B using 
+[Optimum TPU :material-arrow-top-right-thin:{ .external }](https://github.com/huggingface/optimum-tpu){:target="_blank"}
+and [vLLM :material-arrow-top-right-thin:{ .external }](https://github.com/huggingface/optimum-tpu){:target="_blank"}.
```
Comment (Bihan) [inline examples/accelerators/tpu/README.md:17]: Noticed vLLM link is wrong. I will update it.

**Comment by Bihan** [comment]
Closing PR Due to Squashing Complexity and for Repo Cleanliness

This PR is being closed to address issues with squashing commits and maintaining a clean commit history. New [PR](https://github.com/dstackai/dstack/pull/1663) is opened with the appropriate changes consolidated into a single, clean commit.

**Pull request closed without merging**